modules/mobile/app/models/mobile/v0/adapters/immunizations.rb

```diff
@@ -31,6 +31,16 @@ def parse(immunizations)
 
         private
 
+        def date(immunization)
+          date = immunization[:occurrence_date_time]
+          date.presence ? date : StatsD.increment('mobile.immunizations.date_missing') && nil
```

Comment:
I'd use `.present?` instead of `.presence`. Using the `&&` in a ternary also seems strange to me, but I don't think there's anything wrong with it. I tend to be hesitant about using && for control flow because I've been caught before by cases where the first clause returned a falsey value so the second clause doesn't execute. But I don't think that should be an issue here.

Reply:
I typically go for `.present?` as well but the linter was correcting me on it and wanted to do ` date.presence || StatsD.increment('mobile.immunizations.date_missing') && nil ` instead so maybe it is just easier to avoid the ternary all together. For the index.html, yes, I locally reviewed that and it looks good. Got guidance from Jayson on how to represent the nullable field as well to ensure we're consistent.

Reply:
decided to just take out the ternary and put a normal if else statement

Reply:
I guess I can see the linter's point of view. Its suggestion should achieve the same thing, but it does feel less obvious when paired with the && for control flow. I think the fourth option we can add to the list of ways we can handle this is with a guard: 
```
return date if date.present?

StatsD.increment('mobile.immunizations.date_missing')
nil
```

So many options. This might be why some people don't like ruby. :)

Reply:
That's much cleaner. Updated to use that styling but did if `date.blank?` rather then `unless date.present?`. Also did the same for cvx_code.

I updated the documentation to make the new attribute reaction nullable as well to stay consistent with the other attributes.